### DISCUSSION TITLE
Domainstats and freqserver doesn't work

### DISCUSSION THREAD

**Discussion by crysis30300** [Security-Onion-Solutions/securityonion#2140] [Unsupported Versions] [answered]
Hello,

I would like to enable Domainstats and freqserver on SO 2.3.2 version , how can i run those dockers and enable them in kibana , i can find them in docker images but unable to run them!

Thanks for your help

**Comment by weslambert** [comment, marked as answer]
We don't yet fully support Domainstats and Freqserver in Security Onion 2.
